### PR TITLE
Humbug reporter for moonstream crawlers

### PR DESCRIPTION
**Pull request by kompotkot**
## Changes

Humbug reporter for moonstream crawlers.

## How to test these changes?

Tested manually.

### CODE REVIEW

**Comment by kompotkot** [comment]
@bugout-dev check
- [ ] Add env `HUMBUG_REPORTER_CRAWLERS_TOKEN`

File: crawlers/mooncrawl/setup.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
     zip_safe=False,
     install_requires=[
         "moonstreamdb @ git+https://git@github.com/bugout-dev/moonstream.git@39d2b8e36a49958a9ae085ec2cc1be3fc732b9d0#egg=moonstreamdb&subdirectory=db",
+        "humbug",
```
Comment (zomglings) [inline crawlers/mooncrawl/setup.py:37]: Alphabetical order pls (courtesy of @Andrei-Dolgolev ).

File: crawlers/mooncrawl/mooncrawl/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from .reporter import reporter
```
Comment (zomglings) [inline crawlers/mooncrawl/mooncrawl/__init__.py:1]: The reason that you aren't able to install dependencies is that you are importing `reporter` in `__init__.py`, but `setup.py` imports `mooncrawl.version`, which implicitly loads `mooncrawl/__init__.py` which causes `mooncrawl/reporter.py` to load, and `humbug` isn't installed in the environment when you run `setup.py`.

Quick way to solve this is to just hard code version in `setup.py`.

I'm not sure what is the right way to solve it.

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**